src/Management/src/EndpointBase/Hypermedia/HypermediaService.cs

```diff
@@ -49,7 +49,8 @@ public Links Invoke(string baseUrl)
                 {
                     if (!string.IsNullOrEmpty(opt.Id) && !links._links.ContainsKey(opt.Id))
                     {
-                        links._links.Add(opt.Id, new Link(baseUrl + "/" + opt.Path));
+                        var linkPath = $"{baseUrl.TrimEnd('/')}/{opt.Path}";
```

Comment:
```suggestion
                        var linkPath = $"{baseUrl}/{opt.Path}".Replace("//", "/");
```
For consistency with the other two changes

Reply:
^ won't work b/c baseUrl is expected to include `://`

Reply:
`baseUrl` has Http(s?):// in it, so this change won't work here.  In the other place,  it seemed more efficient to do that since no replace occurs if `//` is not found.